### PR TITLE
Artist list updates

### PR DESCRIPTION
**Pull request by rogerhutchings**
- Removes use of localStorage
- Refactors a bunch of stuff
- Gets rid of artists whose sets aren't actually uploaded yet
- Add in Felicia Browne bio ready for data upload

@simoneduca, @shaunanoordin, @eatyourgreens - would someone mind taking a look and merging?

### CODE REVIEW

File: app/modules/set-selector/artists.factory.js [hunk 1]
```diff
@@ -8,13 +8,8 @@ require('./set-selector.module.js')
 // @ngInject
 function ArtistsFactory($q, ArtistListConstants, localStorageService, zooAPIConfig, zooAPI) {
```
Comment (shaunanoordin) [inline app/modules/set-selector/artists.factory.js]: `localStorageService` is no longer used by ArtistsFactory; can it be removed, or is it still required implicitly by `Angular.factory()`/etc?

**Comment by shaunanoordin** [comment]
LGTM!

* Code changes look functional.
* Website compiles & runs normally on localhost

Status: OK; will merge if nobody has any issues to raise.

File: app/modules/set-selector/artists.factory.js (hunk 1 above)
Comment (rogerhutchings) [inline app/modules/set-selector/artists.factory.js]: It's not required - good catch

**Comment by shaunanoordin** [comment]
Status: Updates LGTM! Nice
Actions: Mergin'

**Pull request merged**